test/test_decomp.py

```diff
@@ -382,6 +382,12 @@ def normalize_op_input_output(f, sample, requires_grad=True):
     ("cuda", torch.float16, "nn.functional.cross_entropy"),
 }
 
+# For some operator decompositions, skip check_decomposed() because decompositions may be registered for only one operator variant
+SKIP_CHECK_DECOMPOSED = {
+    "sum",
+    "squeeze",
+}
```

Comment:
Can we revert these skips?

Reply:
Yep, just did it! Forgot to remove these the first update.